### PR TITLE
[NOTEST][RFR][1LP] This PR will change label of smartstate agent.

### PR DESCRIPTION
**Pull request by sbulage**
Changing label of smartstate agent, cleanup job will remove it from respective provider.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8edea7d7d68647cb627b98149f21018cb046486e:

`cfme/tests/cloud_infra_common/test_vm_instance_analysis.py`:
- :bangbang: Line 163:46: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 1]
```diff
@@ -159,9 +159,10 @@ def _hosts_remove_creds():
 
 def set_agent_creds(appliance, request, provider):
     version = appliance.version.vstring
-    agent_data = {"ems": {"ems_amazon": {
-        "agent_coordinator": {"docker_image": "simaishi/amazon-ssa:{}".format(version),
-                              "docker_registry": "docker.io"}}}}
+    docker_image_name = "simaishi/amazon-ssa: {}".format(version)
```
Comment (RonnyPfannschmidt) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: **REQUIRED** why is a space introduced in the docker image name?
**QUESTION** why not let prt run the related test?

Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: As the URL is long and affecting lint, so generated link under `Docker_image_name`.
PRT will failed as SmartState docker creds update require some amount of time, PR #7523 is raised to fix it.

Comment (RonnyPfannschmidt) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: i meant
```diff
- "simaishi/amazon-ssa:{}"
+ "simaishi/amazon-ssa: {}"
``` i believe that is a incorrect change

Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: oops!! I will update it.

**Pull request merged**